guides/common/assembly_importing-content.adoc

```diff
@@ -1,18 +1,4 @@
-[[Importing_Content]]
-= Importing Content
-
-This chapter outlines how you can import different types of {customcontent} to {Project}.
-ifndef::orcharhino[]
-If you want to import RPMs, Files, or different content types to {Project}, it is done with largely the same procedures in this chapter.
-endif::[]
-ifdef::orcharhino[]
-If you want to import RPM or DEB packages, Files, or Puppet Modules to {Project}, it is done with largely the same procedures in this chapter.
-endif::[]
-
-For example, you can use the following chapters for information on specific types of {customcontent} but the underlying procedures are the same:
-
-* xref:Managing_ISO_Images_{context}[]
-* xref:Managing_Custom_File_Type_Content_{context}[]
+include::modules/con_importing-content.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`